combat-trainer.lic

```diff
@@ -392,8 +392,25 @@ class LootProcess
   end
 
   def should_perform_ritual(prepare_consume)
-    !prepare_consume &&
-      ((@necro_train_first_aid && DRSkill.getxp('First Aid') < 32) || DRSkill.getxp('Thanatology') < 32)
+    return false unless @ritual_type
+    return false if prepare_consume
+    return true if @ritual_type == 'dissect' && DRSkill.getxp('First Aid') < 32
+    return true if @ritual_type == 'harvest' && DRSkill.getxp('Skinning') < 32
+    return true if DRSkill.getxp('Thanatology') < 32
+    true
+  end
+
+  def cycle_rituals
```

Comment:
What do you think about renaming this `determine_next_ritual`? I think you could then get rid of `next_ritual` and return directly.

Reply:
i honestly think both names make sense, cycle ritual implies youre cycling, but determine might make the intent more clear